src/App.scss

```diff
@@ -1,8 +1,19 @@
-$primary-bgcolor-rgba: unset; // TODO: need to remove in src
-@import 'config/style/_overides.scss';
-@import 'node_modules/synapse-react-client/dist/template_style/Index.scss';
-@import './portal-components/csbc-home-page/style/all.scss';
-@import 'node_modules/synapse-react-client/dist/style/main.scss';
+@use 'variable_overrides';
+@use 'variables' as Portal;
```

Comment:
We load the variable overrides (highest priority based on load order). Then we load the variables themselves, which have been overridden by the overrides.

Reply:
Does the order matter here? I am just curious because 'variable_overrides' is supposed to override 'variables' right?

Reply:
I _think_ that the Sass compiler is smart enough to determine if a file is being overridden regardless of the order. Ultimately, it shouldn't matter, because `@use` is designed to load each file only once (this contrasts with `@import`, where a file will be loaded each time it is called). 

In my mind, putting the overrides before the variables makes more sense because the overrides are actually setting the values in the 'variables' file, rather than overwriting a global variable.

I am curious to see what happens with something like:

```scss
@use 'variables' as x;
@use 'overrides1';

div.a {
  background-color: x.$color;
}

@use 'overrides2'

div.b {
  background-color: x.$color;
}

```

When would `x.$color` take on the value of `$color` in `overrides1.scss` or `overrides2.scss`?

Reply:
Got it now! I'd just make sure the default css in `'variables' as Portal` doesn't overwrite the ones in `variable_overrides`.